interface/patient_file/encounter/forms.php

```diff
@@ -770,8 +757,7 @@ function myGetRegistered($state = "1", $limit = "unlimited", $offset = "0")
 <?php if ($GLOBALS['enable_follow_up_encounters']) { ?>
     <a href='#' class='btn btn-primary' onclick='return createFollowUpEncounter()'><?php echo xlt('Create follow-up encounter') ?></a>
 <?php } ?>
-&nbsp;&nbsp;&nbsp;<a href="#" onClick='expandcollapse("expand");' style="font-size:80%;"><?php echo xlt('Expand All'); ?></a>
-&nbsp;&nbsp;&nbsp;<a  style="font-size: 80%;" href="#" onClick='expandcollapse("collapse");'><?php echo xlt('Collapse All'); ?></a>
+&nbsp;&nbsp;&nbsp;<button type="button" class="btn btn-primary btn-sm" data-toggle="collapse" data-target="#demo"><?php echo xlt('Expand / Collapse'); ?></button>
```

Comment:
can you please replace those `&nbsp`  with padding/margin. I know it's not part of your PR but they smell bad.